scripts/d.polar/d.polar.html

```diff
@@ -71,8 +71,3 @@ <h2>AUTHORS</h2>
 Markus Neteler, ITC-irst, Italy<br>
 Bruno Caprile, ITC-irst, Italy<br>
 Hamish Bowman, Otago University, New Zealand<br>
-
-<!--
-<p>
-<i>Last changed: $Date$</i>
--->
```

Comment:
`<h2>XGRAPH REQUIREMENT</h2>` and `<h2>REFERENCES</h2>` section should be before `<h2>SEE ALSO</h2>` section.